### PR TITLE
Add support for colorToHex util on IE11

### PR DESCRIPTION
**Pull request by jmagalhaes-xing**

### CODE REVIEW

File: src/app/core/utils/color.util.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+  let colorValue;
+
+  cssVars({
+    onComplete: (cssText, styleNodes, cssVariables, benchmark): void => {
+      colorValue = cssVariables[`--${color.toLowerCase()}`];
+    },
+    rootElement: document,
+  });
+
+  return colorValue || getComputedStyle(document.body).getPropertyValue(`--${color.toLowerCase()}`).trim();
```
Comment (tiago) [inline src/app/core/utils/color.util.ts]: Isn't the `|| getComputedStyle(...` part redundant now? The polyfill should work on both old a new browsers.

Comment (jmagalhaes-xing) [reply src/app/core/utils/color.util.ts]: Nope, unfortunately no. I haven't dig deep into the tool but it only works on IE. For other browsers, I need the "normal" way to get the var

File: src/app/core/utils/color.util.ts [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+import cssVars from 'css-vars-ponyfill';
+
 import { Core } from '../typings';
 
 export function colorToHex(color: Core.Colors): string {
-  return getComputedStyle(document.body).getPropertyValue(`--${color.toLowerCase()}`);
+  let colorValue;
+
+  cssVars({
+    onComplete: (cssText, styleNodes, cssVariables, benchmark): void => {
```
Comment (tiago) [inline src/app/core/utils/color.util.ts]: Given that you're only using `cssVariables`, maybe remove `benchmark` and prefix others with the `_` convention to signal they are being unused: `onComplete: (_cssText,_ styleNodes, cssVariables): void => {`

Comment (jmagalhaes-xing) [reply src/app/core/utils/color.util.ts]: Done

**Pull request merged**